src/plugins/auto/executable_network.cpp

```diff
@@ -239,6 +239,8 @@ MultiDeviceExecutableNetwork::MultiDeviceExecutableNetwork(const std::string&
                       std::call_once(_firstLoadOC, [this] () {
                               _firstLoadPromise.set_value();
                               });
+                      if (_loadContext[CPU].isEnabled && _loadContext[ACTUALDEVICE].isAlready)
+                          _recycleCond.notify_one();
```

Comment:
still need to lock  _recycleCond.notify_one();

Reply:
fixed